src/proc/rates-printer.cpp

```diff
@@ -77,7 +77,7 @@ namespace librealsense
         auto curr_time = std::chrono::steady_clock::now();
         _time_points.push_back(curr_time);
         auto oldest = _time_points[0];
-        if (_time_points.size() > _stream_profile.fps())
+        if (_time_points.size() > size_t(_stream_profile.fps()))
             _time_points.erase(_time_points.begin());
```

Comment:
Please choose between c cast - (size_t)(_stream_profile.fps()) - or static_cast

Reply:
I would tend to c++, but in those situations it becomes too verbose.

Reply:
Done